tests/acceptance/editor-sections-test.js

```diff
@@ -167,6 +167,20 @@ test('hitting enter at end of a section creates new empty section', (assert) =>
                     offset: 0});
 });
 
+Helpers.skipInPhantom('deleting across 0 sections does nothing if editing is disabled', (assert) => {
```

Comment:
this should be safe to run in phantom, and should say "delete across 2 sections"